### PR TITLE
ticket-4-show-list-all-articles

### PR DESCRIPTION
**Pull request by Hydrochilli**
adds list of card items for each article with url route to /article and with functional , minimal,styling to improve later.

### CODE REVIEW

**Comment by nc-niamh-smith** [comment]
resubmit with the axios request to the api instead of localhost :) Also, feel free to merge once you have sent a PR to us!

File: src/App.jsx [hunk 1]
```diff
@@ -1,34 +1,26 @@
...
-        </p>
-      </div>
-      <p className="read-the-docs">
-        Click on the Vite and React logos to learn more
-      </p>
-    </>
+    <BrowserRouter>
+      <Header />
+      <Routes>
+        <Route path="/articles" element={<ArticlesPage/>} />
```
Comment (nc-rayhaan) [inline src/App.jsx:18]: I knew this was the path after looking at the code, but a regular user wouldn't! Please make the articles to render on path `/` or have a home page which links to `/articles`

File: src/Components/ArticlesItem.jsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+
+function ArticleItem({ article}) {
+    console.log('Rendering ArticleItem with article:', article)
```
Comment (nc-rayhaan) [inline src/Components/ArticlesItem.jsx:5]: Please remove any console logs before pushing

File: src/Components/ArticlesPage.jsx [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+            
+               setArticles(response.data.articles)
+              
+                setLoading(false)
+            } catch (err) {
+                
+                setError(err.message)
+                setLoading(false)
+            }
+        }
```
Comment (nc-rayhaan) [inline src/Components/ArticlesPage.jsx:24]: This does work, but would be a lot cleaner if your api functions are in a separate file. Personal preference though

**Review by nc-rayhaan — COMMENTED** [review]
Nice work!

**Pull request merged**